Fix Battle Music/Pokemon Cries settings  & add option to hide banner

turbedi opened the pull request:
Now the settings for Pokemon Cries and Battle Music are separated, so
you can now have only Pokemon Cries.

Also added an option to hide announcement banner (requested by Nightmare Moon).

Wanted to separate the PRs, but I accidentally updated the wrong branch and I am too lazy to fix this.

Fuzzysqurl commented:
Change [Line 135](https://github.com/po-devs/pokemon-online/pull/1442/files?diff=split#diff-6869c20197d69111f3873dea62c352b7R135) to ``mylayout->addWidget(getSceneWidget(), 0, 0, 1, 5);`` and it should look better possibly.

There might be one around for increasing the width of the battle window and of the moves selection, but I'm too lazy to dig atm. If that one change doesnt fix it, then you gotta find the other two.

Also what about Old Battle Window? Or does this reuse the code? I can't remember.

turbedi commented:
I've changed Line 135 as you said and it looks much better now: http://puu.sh/mXPMT.jpg
Do you know how to stretch the move buttons too? I think if this is also fixed, the PR is good to go.

And about the Old Battle Window: Yes, it reuses the code, so it now also has the bug with the move buttons not being stretched properly.

turbedi commented:
I've managed to fix the remaining issue on my own.
